configure.ac

```diff
@@ -62,14 +62,13 @@ getpeername(0,0,&len);
 dnl Android requires -llog for printf and friends
 dnl and will only run position independent executables
 AC_MSG_CHECKING([if building on Android])
-AC_TRY_COMPILE([
+AC_TRY_COMPILE([], [
   #ifdef __BIONIC__
   int ok;
   (void)ok;
   #else
   choke me
   #endif],
-[func("a"); func("a", "b"); func("a", "b", "c")],
```

Comment:
This seems to have just been a copy/paste from another test.  I can't see how this test could have ever worked right, though, since the body of the `main()` needs to be the second arg to `AC_TRY_COMPILE()`.  The `(void)ok;` would have ended up outside of the function body even on Adroid!